### PR TITLE
Simple Attachment Storage implementation using Requery/H2 database

### PR DESCRIPTION
**Pull request by polybioz**
Simple Attachment Storage implementation of AttachmentStorage interface using Requery.

### CODE REVIEW

**Comment by josecoll** [comment]
Please rebase with master before I review (as there is a lot of duplicate code).
Thanks.

**Comment by josecoll** [comment]
In general, please also remove empty lines.
And use Kotlin null safety checking rather than (if X == null) 
See https://kotlinlang.org/docs/reference/null-safety.html

File: build.gradle [hunk 1]
```diff
@@ -10,7 +10,7 @@ buildscript {
     // Dependency versions. Can run 'gradle dependencyUpdates' to find new versions of things.
     //
     // TODO: Sort this alphabetically.
-    ext.kotlin_version = '1.0.5-2'
+    ext.kotlin_version = '1.0.6'
```
Comment (Clintonio) [inline build.gradle]: Is this strictly necessary for this PR? I'd prefer to keep changes like this in their own PR if possible.

Comment (mikehearn) [inline build.gradle]: It'll go away after the rebase. It's a part of the requery PR that was just merged, and yeah, not ideal but it's not a big deal either. Jose asked me and I said it was fine.

**Comment by mikehearn** [comment]
Hi Karel, can you rebase this onto master?

**Comment by polybioz** [comment]
Recreated the pull request after strange issues with rebasing.
